src/main/java/com/incognia/FeedbackEvent.java

```diff
@@ -5,6 +5,10 @@
 public enum FeedbackEvent {
```

Comment:
What you think about reordering the enums to follow the order in our [docs](https://developer.incognia.com/docs/br/v6/apis/feedback/feedback-api#understanding-event-types)?

Reply:
shouldn't be a problem

Reply:
done